game/static/game/css/landing.css

```
@@ -361,41 +362,91 @@ body {
+       Each layer uses a different blur radius to mimic a real light source. */
+    box-shadow:
+        0 0 8px 2px rgba(255, 224, 130, 0.95),
+        0 0 18px 6px rgba(240, 192, 64, 0.7),
+        0 0 36px 12px rgba(240, 192, 64, 0.35),
+        0 0 60px 20px rgba(240, 192, 64, 0.15);
+    /* Breathing animation lives on the parent here because we only animate
+       opacity (not transform), so it does not compete with JS translate3d. */
+    animation: cursor-Glow-Breathe 1.6s ease-in-out infinite;
+}
+
+@keyframes cursor-Glow-Breathe {
```

Stylelint (17.13.0)

[error] Expected keyframe name "cursor-Glow-Breathe" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)